transformer_engine/jax/csrc/modules.cpp

```diff
@@ -1488,5 +1487,265 @@ void CrossFusedAttnBackward(cudaStream_t stream, void **buffers, const char *opa
     nvte_tensor_pack_destroy(&aux_input_tensors);
 }
 
+pybind11::tuple GetFusedAttnForwardWorkspaceSizes(
```

Comment:
The only difference I can see between this function and `GetCrossFusedAttnForwardWorkspaceSizes()` is the choice of `qkv_layout`. Is that correct?

If so, we should combine all the workspace functions into a single one, which would also make it much easier to eliminate duplicate code in the JAX Python primitives.